test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -2030,8 +2030,8 @@ public void Defect_MAGN_491_DS()
             ConnectorViewModel secondConnector = connectors[1];
 
             // Find out the corresponding ports they connect to.
-            Point2D firstPoint = firstConnector.ConnectorModel.End.Center;
-            Point2D secondPoint = secondConnector.ConnectorModel.End.Center;
+            Point2D firstPoint = firstConnector.Model.End.Center;
+            Point2D secondPoint = secondConnector.Model.End.Center;
```

Comment:
how come these have changed?

Reply:
This had to do with the resharper issue that went through and renamed all things `model` with `connectorModel`.. just reverted it for clarity.

Reply:
This change should have been picked up in the main PR. Will do that now.

Reply:
Changed this back in this PR and in the original PR it has been changed correctly to say `model`.